### DISCUSSION TITLE
Binding: Wert eines anderen DP anzeigen

### DISCUSSION THREAD

**Discussion by hosabeni** [Zefau/ioBroker.jarvis#2590] [Q&A]
Version: 3.2.0-alpha.59

Ist es auch möglich unter Anzeigevariante je nach Bedingung den Wert eines anderen Datenpunkt anzuzeigen. 
Wie hier im Beispiel den Wert vom `{fb-checkpresence.0.fb-devices.IOBroker.WLAN.rx_rcpi} ` wenn die Bedingung `"{fb-checkpresence.0.fb-devices.IOBroker.active} == true && {val} == '802.11'"` erfüllt ist. Aktuell wird `{fb-checkpresence.0.fb-devices.IOBroker.WLAN.rx_rcpi}` als Text angezeigt bzw. aktuell wird nichts angezeigt und Label und Icon werden falsch dargestellt

``` 
{
 "{fb-checkpresence.0.fb-devices.IOBroker.active} == true && {val} == 'Ethernet'": "LAN",
 "{fb-checkpresence.0.fb-devices.IOBroker.active} == true && {val} == '802.11'": "{fb-checkpresence.0.fb-devices.IOBroker.WLAN.rx_rcpi}"
}
```

Hat sicher erledigt. 
Hab meinen Fehler gefunden. Sorry.

**Comment by mcuiobroker** [comment]
Welchen Fehler hast du gefunden? Wie gelöst?
